api/desecapi/metrics.py

```diff
@@ -1,54 +1,43 @@
 from prometheus_client import Counter, Histogram, Summary
 
-metrics = {}
 
+class Model:
+    captcha_content_created = Counter('desecapi_captcha_content_created', 'number of times captcha content created',['kind'])
+    autodelegation_created = Counter('desecapi_autodelegation_created', 'number of autodelegations added')
+    autodelegation_deleted = Counter('autodelegation_deleted', 'number of autodelegations deleted')
```

Comment:
Counter name is missing `desecapi_` prefix.

The previous structure had the advantage of not duplicating the name (i.e. using the same thing on the lhs and rhs). I had made that choice a) to ensure consistency, and b) that you can infer the metric name _as it is exposed in Prometheus_ from the code line where you increment the counter. That connection is not enforced in the new structure.

Before I double-check all entries for completeness and spelling: Have you done the refactoring / reformatting in an automated fashion, or should I be in the "proofread a human" mindset?

Reply:
oh I see. Well let's exclude this commit then for now.